### PR TITLE
initial draft of terms, event guidelines and package structure

### PR DESCRIPTION
**Pull request by dimitriirtimid**

### CODE REVIEW

File: functionality/glossary.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Glossary
+
+These terms are commonly used in LocalMotion. Avoid synonyms when referring to these concepts.
+- Initiative: Collaboration by a group of people to reach a common goal. In LocalMotion the goal is typically bound to a location.
+- Member: A person who collaborates in an Initiative is a member of that initiative.
+- Playground: A specific type of Initiative, namely a smokefree-playground initiative. Use this term only when referring to an initiative in this specific
+context, otherwise use Initiative. 
+- Manager: a specific type of member in an Initiave which more rights to issue commands. Currently this is not a generic concept and therefore only
```
Comment (AnandAili) [inline functionality/glossary.md]: typo mistake 'Initiave'

File: technical/code_structuring.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Code structuring
+
+## Standards for Java packages
+
+All packages in LocalMotion must be subpackages of `io.localmotion`.
```
Comment (AnandAili) [inline technical/code_structuring.md]: Do we have any specific reason for a package name to start with 'io'?. java naming convention for package name https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

File: technical/code_structuring.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- io.localmotion.application
+- io.localmotion.chatbox
+- io.localmotion.configuration
+- io.localmotion.personaldata
+- io.localmotion.infrastructure.aws.rds.secretmanager
+- io.localmotion.infrastructure.aws.cognito
+- io.localmotion.infrastructure.database
+- io.localmotion.infrastructure.axon
+- io.localmotion.infrastructure.graphql
+- io.localmotion.infrastructure.axon
```
Comment (AnandAili) [inline technical/code_structuring.md]: repeated

File: technical/code_structuring.md (hunk 2 above)
Comment (dimitriirtimid) [reply technical/code_structuring.md]: we have the localmotion.io domain, so I propose to use io.localmotion... as package name

**Pull request merged**